### ISSUE TITLE
shuffle_particles doesn't correctly handle floppy bodies that are also Gaussians

### ISSUE THREAD

**Issue by cgreenberg** [salilab/pmi#106]
We only just noticed this in the release version, which has additional checks. It tries to transform them with:

``` python
IMP.core.transform(IMP.core.XYZ(fb), transformation)
```

But it needs to check if they're rb's, and if so do:

``` python
IMP.core.transform(IMP.core.RigidBody(fb), transformation)
```

**Comment by benmwebb** [comment]
This is a usage check... so the only way you should miss that is if you're running in fast mode. And surely you're not testing your protocol in fast mode? There are no checks at all there...

**Comment by Pellarin** [comment]
charles was this fixed?

**Comment by cgreenberg** [comment]
I don't believe so. I'll work on a test.

**Comment by cgreenberg** [comment]
This was finally fixed with 5082317541879aa3ec7e4a29e11d575879be37ab
